index.js

```diff
@@ -1,5 +1,7 @@
 try {
   require('dotenv').load();
-} catch (e) { }
```

Comment:
Similarly, let's ask @Pinpickle What the intention here was

Reply:
done